torctl

```diff
@@ -98,6 +99,28 @@ check_root() {
     fi
 }
 
+check_tor() {
+if tor -h &> /dev/null ; then
+        sleep 0.1
+else info "Tor is not installed on your system. Installation started."
+     if pacman -Sy tor --noconfirm &> /dev/null ; then
+        sleep 0.1
+     else err "Tor installation failed."
+     fi
```

Comment:
Please remove this for several reasons:

1) No need to use tor -h to see if it's or no instaled, use `command -v tor` instead.
2) Not everyone whoi use this script has ArchLinux or Arch based distros.
3) `pacman -Sy` is one of the worst things you can do in an ArchLlinux system, it will lead to [partial upgrades](https://wiki.archlinux.org/index.php/System_maintenance#Avoid_certain_pacman_commands)